lib/graphql/utils.ex

```diff
@@ -113,4 +124,8 @@ defmodule BridgeEx.Graphql.Utils do
     |> LanguageConventions.to_internal_name(:read)
     |> String.to_atom()
   end
+
+  # https://elixirforum.com/t/creating-list-adding-elements-on-specific-conditions/6295/4?u=learts
```

Comment:
Just a personal opinion, but IMHO this link could be removed, as the code is very clear by itself :)